frontend/components/FeatureList.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div class="flex flex-col w-full space-y-10">
+        <div v-for="feature in features" :key="feature.name">
+            <TextBlock :title="feature.title">
+                <div class="mb-4">
+                    <span class="text-gray-700 dark:text-gray-200">URL:</span>
+                    <a v-if="feature.url" :href="feature.url"> {{ feature.url }} </a>
+                    <span v-else class="italic">Not yet implemented</span>
+                </div>
+                <h3 class="text-lg text-gray-700 font-semibold dark:text-gray-200 my-2">List of features:</h3>
+                <ul
+                    v-for="(point, index) in feature.features"
+                    :key="index"
+                    class="list-inside list-disc text-gray-700 dark:text-gray-200"
+                >
+                    <li :class="`Indent-${point.indent}`">{{ point.text }}</li>
```

Comment:
- This is not correct from the semantic point of view. The person with a screen reader would have a degraded possibility to understand the lists. But to implement it properly, you would need to 1) put `ul` into a separate component 2) [provide name of the component](https://github.com/sidestream-tech/unified-auctions-ui/blob/dd6882ad6cb84ee54d95b8e5bd933a31ec61a702/frontend/components/Auction.vue#L257) 3) call it recursively (you may want to restructure the FeatureList as well)
- Second reason to refactor this: class names constructed dynamically will be removed in production build https://stackoverflow.com/a/69687876